examples/knative/powercli/kn-pcli-tag/handler.ps1

```diff
@@ -71,6 +71,16 @@ Function Process-Handler {
    # Extract VM Name from event
    $vmName = $cloudEventData.Vm.Name
 
+   # Ensure vCenter session is still viable for Tag actions (This expires before the general session)
+   Try {
+      Get-TagCategory -ErrorAction:stop | Out-Null
+      Write-Host "$(Get-Date) - vCenter session verified viable for tagging."
+   } Catch {
+      Write-Host "$(Get-Date) - vCenter session for tagging has expired. Reinitializing..."
+      Disconnect-VIServer -Server * -Confirm:$false -Force:$true
+      Process-Init
```

Comment:
What if `Process-Init` throws again? The current behavior would be that the handler returns, the exception would be handled by the server.ps try/catch and return INTERNAL_SERVER_ERROR to the caller (e.g. broker). A broker would generally retry, potentially leading to a (temporary) loop.

Either we accept this (until we return better/typed errors to the server.ps code) or we exit 1 here to crash the function.